px-data-grid.html

```diff
@@ -2199,7 +2204,7 @@
           } else {
             this._sorters.forEach(sorter => {
               sorter._order = null;
-              sorter.direction = null;
+              sorter.direction = sorter.defaultSortDirection || null;
```

Comment:
Better solution: set to null (as before), but only do this forEach loop in the following `if(sorted.direction)` statement, which is what it should have been all along.